unicodetools/src/test/java/org/unicode/text/UCD/TestCodeInvariants.java

```diff
@@ -41,12 +44,24 @@ public static void main(String[] args) {
         testGcbInDecompositions(true);
     }
```

Comment:
The main() function should go away in the JUnit test, right?

Reply:
removed